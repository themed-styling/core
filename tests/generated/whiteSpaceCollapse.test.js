import 'jest-styled-components'

import { whiteSpaceCollapse } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(whiteSpaceCollapse, 'whiteSpaceCollapse')
