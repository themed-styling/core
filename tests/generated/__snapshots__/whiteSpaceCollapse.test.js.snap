// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`whiteSpaceCollapse renders 0 1`] = `
Array [
  .c0 {
  white-space-collapse: 0;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: 0;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: 0;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`whiteSpaceCollapse renders function 1`] = `
Array [
  .c0 {
  white-space-collapse: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: has;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: has;
}

<div
    className="c0"
  />,
]
`;

exports[`whiteSpaceCollapse renders invalid theme path 1`] = `
Array [
  .c0 {
  white-space-collapse: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`whiteSpaceCollapse renders negative number 1`] = `
Array [
  .c0 {
  white-space-collapse: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`whiteSpaceCollapse renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  white-space-collapse: July;
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    white-space-collapse: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: July;
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    white-space-collapse: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    white-space-collapse: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    white-space-collapse: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`whiteSpaceCollapse renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    white-space-collapse: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    white-space-collapse: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    white-space-collapse: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    white-space-collapse: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    white-space-collapse: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    white-space-collapse: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    white-space-collapse: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`whiteSpaceCollapse renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  white-space-collapse: July;
}

@media May {
  .c0 {
    white-space-collapse: August;
  }
}

@media April {
  .c0 {
    white-space-collapse: September;
  }
}

@media January {
  .c0 {
    white-space-collapse: November;
  }
}

@media April {
  .c0 {
    white-space-collapse: August;
  }
}

@media June {
  .c0 {
    white-space-collapse: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: July;
}

@media May {
  .c0 {
    white-space-collapse: August;
  }
}

@media April {
  .c0 {
    white-space-collapse: September;
  }
}

@media January {
  .c0 {
    white-space-collapse: November;
  }
}

@media April {
  .c0 {
    white-space-collapse: August;
  }
}

@media June {
  .c0 {
    white-space-collapse: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    white-space-collapse: November;
  }
}

@media April {
  .c0 {
    white-space-collapse: January;
  }
}

@media January {
  .c0 {
    white-space-collapse: December;
  }
}

@media April {
  .c0 {
    white-space-collapse: April;
  }
}

@media June {
  .c0 {
    white-space-collapse: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    white-space-collapse: November;
  }
}

@media April {
  .c0 {
    white-space-collapse: January;
  }
}

@media January {
  .c0 {
    white-space-collapse: December;
  }
}

@media April {
  .c0 {
    white-space-collapse: April;
  }
}

@media June {
  .c0 {
    white-space-collapse: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`whiteSpaceCollapse renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    white-space-collapse: July;
  }
}

@media April {
  .c0 {
    white-space-collapse: August;
  }
}

@media January {
  .c0 {
    white-space-collapse: September;
  }
}

@media April {
  .c0 {
    white-space-collapse: November;
  }
}

@media June {
  .c0 {
    white-space-collapse: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    white-space-collapse: July;
  }
}

@media April {
  .c0 {
    white-space-collapse: August;
  }
}

@media January {
  .c0 {
    white-space-collapse: September;
  }
}

@media April {
  .c0 {
    white-space-collapse: November;
  }
}

@media June {
  .c0 {
    white-space-collapse: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    white-space-collapse: January;
  }
}

@media April {
  .c0 {
    white-space-collapse: December;
  }
}

@media January {
  .c0 {
    white-space-collapse: April;
  }
}

@media April {
  .c0 {
    white-space-collapse: May;
  }
}

@media June {
  .c0 {
    white-space-collapse: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    white-space-collapse: January;
  }
}

@media April {
  .c0 {
    white-space-collapse: December;
  }
}

@media January {
  .c0 {
    white-space-collapse: April;
  }
}

@media April {
  .c0 {
    white-space-collapse: May;
  }
}

@media June {
  .c0 {
    white-space-collapse: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`whiteSpaceCollapse renders postitive number 1`] = `
Array [
  .c0 {
  white-space-collapse: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`whiteSpaceCollapse renders string 1`] = `
Array [
  .c0 {
  white-space-collapse: July;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: July;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: June;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: June;
}

<div
    className="c0"
  />,
]
`;

exports[`whiteSpaceCollapse renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`whiteSpaceCollapse renders valid theme path 1`] = `
Array [
  .c0 {
  white-space-collapse: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  white-space-collapse: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
